### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by phantomega** [community/community#129268] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Security and Privacy

### Body

My account is suspended but i do not receive any email from github indicating why my account got suspended. And when i tried to login to my account using github support, I do not receive any 2-FA code. Is this because I am working in a country with ties to a communist country?

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @phantomega ! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

\The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
